app/controllers/provider_interface/activity_log_controller.rb

```diff
@@ -15,12 +15,15 @@ def index
         end
       end
 
-      provider_ids = current_provider_user.providers.ids.join(', ')
-      if benchmark.real.between?(30, 115)
-        Rails.logger.info "Activity page slow for providers [#{provider_ids}] took #{benchmark.real.to_i} seconds"
-      end
-      if benchmark.real > 115
-        Rails.logger.info "Activity page timed out for providers [#{provider_ids}] took #{benchmark.real.to_i} seconds"
+      if benchmark.real > 30
+        provider_ids = current_provider_user.providers.ids.join(', ')
```

Comment:
If we don't `join` these, would they get sent to Logit as an Array and allow us to filter the logs based on the array items?

Reply:
To note I think we should only store the ids in the array